ntbs-service/Helpers/RouteHelper.cs

```diff
@@ -1,25 +1,29 @@
-namespace ntbs_service.Helpers
+﻿namespace ntbs_service.Helpers
 {
     public static class RouteHelper
     {
-        public static string GetFullNotificationEditPath(string subPath, int id, bool isBeingSubmitted)
-        {
-            return $"{GetNotificationEditBasePath(subPath)}?id={id}&isBeingSubmitted={isBeingSubmitted}";
-        }
-
-        public static string GetNotificationEditBasePath(string subPath)
-        {
-            return $"/Notifications/Edit/{subPath}";
-        }
+        public static string NotificationBasePath => "/Notifications/{0}/{1}";
 
-        public static string GetFullNotificationPath(string subPath, int id)
+        public static string GetNotificationPath(string subPath, int id, bool isBeingSubmitted = false)
         {
-            return $"{GetNotificationBasePath(subPath)}?id={id}";
+            var pathWithQueryString = string.Concat(subPath, isBeingSubmitted ? "?isBeingSubmitted=True" : string.Empty);

```

Comment:
I think you could use the QueryHelpers class (https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.webutilities.queryhelpers?view=aspnetcore-2.2) here instead of building the query string yourself: `QueryHelpers.AddQueryString(subPath, "isBeingSubmitted", True)`